packages/garbo/src/tasks/yachtzee/index.ts

```diff
@@ -0,0 +1,125 @@
+import { inebrietyLimit, myInebriety, use, useSkill } from "kolmafia";
+import {
+  $effect,
+  $item,
+  $items,
+  $location,
+  $skill,
+  AprilingBandHelmet,
+  CinchoDeMayo,
+  Delayed,
+  get,
+  have,
+} from "libram";
+import { bestFamUnderwaterGear, bestYachtzeeFamiliar } from "./familiar";
+import { getBestWaterBreathingEquipment } from "./outfit";
+import {
+  freeFishyAvailable,
+  freeNCs,
+  maximumYachtzees,
+  shouldYachtzee,
+} from "./lib";
+import { GarboStrategy, Macro } from "../../combat";
+import { GarboTask } from "../engine";
+import { willDrunkAdventure } from "../../lib";
+import { globalOptions } from "../../config";
+
+function doYachtzeeTask(additionalReady: () => boolean) {
+  return {
+    completed: () => !get("noncombatForcerActive"),
+    ready: () => additionalReady() && have($effect`Fishy`),
+    do: $location`The Sunken Party Yacht`,
+    outfit: () => {
+      const overdrunk = myInebriety() > inebrietyLimit();
+      const yachtzeeFamiliar = bestYachtzeeFamiliar();
+      const modifiers = ["20 Meat"];
+      if (
+        !(
+          yachtzeeFamiliar.underwater ||
+          have($effect`Driving Waterproofly`) ||
+          have($effect`Wet Willied`)
+        )
+      ) {
+        modifiers.push("underwater familiar");
+      }
+      const equips = [
+        getBestWaterBreathingEquipment(freeNCs()).item,
+        bestFamUnderwaterGear(yachtzeeFamiliar),
+      ];
+      if (overdrunk) equips.push($item`Drunkula's wineglass`);
+      return {
+        equip: equips,
+        modifier: modifiers,
+        avoid: $items`anemoney clip, cursed magnifying glass, Kramco Sausage-o-Matic™, cheap sunglasses`,
+        familiar: yachtzeeFamiliar,
+      };
+    },
```

Comment:
This code can be simplified by using an `Outfit` instance, instead of just returning a spec imo